README.md

```diff
@@ -1,61 +1,52 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# capstone project - Conference page
 
-# Project Name
-
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![screenshot](https://github.com/mohamedelshamy55/firstcapstone/blob/mohamedelshamy55-patch-1/screenshot1.png)
+![screenshot](https://github.com/mohamedelshamy55/firstcapstone/blob/mohamedelshamy55-patch-1/screenshot2.png)
+### Deployment
+- https://mohamedelshamy55.github.io/firstcapstone/
 
 ## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
+- HTML
+- CSS
+- JavaScript
+- 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+**Porject's features are added into seperate branch to keep main branch safe.**
 
 
 To get a local copy up and running follow these simple example steps.
 
+- create a new folder
+- acccess the folder via command line
+- run this under the command line: https://github.com/mohamedelshamy55/firstcapstone/tree/mohamedelshamy55-patch-1
+
 ### Prerequisites
+-Html
+-CSS
+-Git
 
 ### Setup
+-None
+
 
 ### Install
+-Git
 
 ### Usage
+-None
 
 ### Run tests
+-None
```

Comment:
- [ ] In this section you should try and be a bit more specific what are the commands a person should use in order to run your project _(keep in mind that not everyone has coding experience)_. You can use [my readme file](https://github.com/VanessaAoki/JS-Capstone/blob/development/README.md) as a reference, if you want.
> If there are no tests to run, it's better to just leave it out of the file instead of having "none".